e2e-tests/data/invalid-view.json

```diff
@@ -280,7 +280,7 @@
     "grid": {
       "columnSizes": "1fr 3px 3fr 3px 1fr",
       "leftComponentBottom": "SimulationPanel",
-      "leftComponentTop": "ActivityTypesPanel",
+      "leftComponentTop": "ActivityAndResourceTypesPanel",
```

Comment:
Could rename this to something more generic to minimize breaking changes to view definitions? Like "TimelineItemsPanel" or "BrowsePanel"?

Reply:
I think I like "TimelineItemsPanel".